ui.apps/src/main/content/jcr_root/apps/acs-commons/config/com.adobe.acs.commons.users.impl.EnsureServiceUser-email-service.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Let's rename the file to EnsureServiceUser-EmailService.xml to be consistent with other files.

Reply:
Done. 👍